litho-processor/src/main/java/com/facebook/litho/specmodels/processor/MountSpecModelFactory.java

```diff
@@ -109,6 +109,7 @@ public MountSpecModel create(
       @Nullable DependencyInjectionHelper dependencyInjectionHelper,
       @Nullable InterStageStore interStageStore) {
     return new MountSpecModel(
+        element,
```

Comment:
I wonder if it's still useful to pass in the other arguments that are directly derived from `element`.

Reply:
@passy I was wondering exactly the same thing! We could also then create separate `ElementUtils` that would contain info extraction methods, replacing essentially all of those assignments in `*SpecModel` constructors. This would though imply a bigger refactor! If this is something that the Litho team would feel ok with, I would be more than happy to do it. 

I believe thought that this would only apply to `*SpecModel` implementing classes and not `TestSpecModel` or `MockSpecModel` which do not contain an originating element in purpose.